extensions/braze/braze-currents-users-event.schema.json

```diff
@@ -0,0 +1,301 @@
+          "title": "App ID",
+          "type": "string",
+          "description": "API ID of the app this action occurred on"
+        },
+        "braze:timezone": {
+          "title": "Timezone",
+          "type": "string",
+          "description": "timezone of the user"
+        },
+        "braze:user": {
+          "title": "User",
+          "type": "object",
+          "properties": {
+            "braze:userId": {
+              "title": "User ID",
+              "type": "string",
+              "description": "BSON id of the user that performed the event",
+              "isRequired": true
+            },
+            "braze:externalUserId": {
+              "title": "External User ID",
+              "type": "string",
+              "description": "external user id of the user"
+            },
+            "braze:emailAddress": {
+              "title": "Email Address",
+              "type": "string",
+              "description": "email address of the user"
+            }
+          },
+          "required": [
+            "braze:userId"
+          ]
+        },
+        "braze:device": {
+          "title": "Device",
+          "type": "object",
+          "properties": {
+            "braze:platform": {
+              "title": "Platform",
+              "type": "string",
+              "description": "platform of the device"
+            },
+            "braze:osVersion": {
+              "title": "OS Version",
+              "type": "string",
+              "description": "version of the operating system of the device"
+            },
+            "braze:deviceModel": {
+              "title": "Device Model",
+              "type": "string",
+              "description": "model of the device"
+            },
+            "braze:deviceId": {
+              "title": "Device ID",
+              "type": "string",
+              "description": "id of the device on which the custom event occurred"
+            }
+          }
+        },
+        "braze:customEvent": {
+          "title": "Custom Event",
+          "type": "object",
+          "properties": {
+            "braze:name": {
+              "title": "Name",
+              "type": "string",
+              "description": "name of the custom event"
+            },
+            "braze:properties": {
+              "title": "Properties",
+              "type": "string",
+              "description": "custom properties of the event stored as a JSON encoded string"
+            }
+          }
+        },
+        "braze:campaign": {
+          "title": "Campaign",
+          "type": "object",
+          "properties": {
+            "braze:campaignId": {
+              "title": "Campaign ID",
+              "type": "string",
+              "description": "API ID of the campaign this email belongs to"
+            },
+            "braze:campaignName": {
+              "title": "Campaign Name",
+              "type": "string",
+              "description": "name of the campaign"
```

Comment:
starts with a capital letter " Name of the campaign." description always ends with a period(.)